_topic_map.yml

```diff
@@ -129,12 +129,12 @@ Topics:
 #    File: installing-openstack-installer
   - Name: Installing a cluster on OpenStack with customizations
     File: installing-openstack-installer-custom
+  # - Name: Load balancing deployments on OpenStack
```

Comment:
Are you pulling this content from the release?

Reply:
Yes.

Reply:
It's possible that someone might want it back in the near term, but we don't think it's necessary to include it for now. @kalexand-rh